tests/pytests/common.py

```diff
@@ -370,3 +379,92 @@ def expect_when(self, cond_val, func: Callable[[Query], Any]):
         if cond_val == self.cond_val:
             func(self.env.expect(*self.query))
         return self
+
+def module_path():
+    # path/to/RediSearch/tests/pytests
+    root_path =  dirname(dirname(dirname(os.path.realpath(__file__))))
+    search_file_path = join(root_path, 'bin', f'{OS}-{ARCH}-{BUILD_MODE}', 'search', 'redisearch.so' )
+    json_file_path = join(root_path, 'bin', f'{OS}-{OSNICK}-{ARCH}', 'RedisJSON', 'rejson.so' )
```

Comment:
`runtests.sh` has `MODULE` and `REJSON_PATH` env vars for module paths. Those can we exported to be available for tests.

Reply:
what about `ROOT` directory?